module/zfs/dmu_objset.c

```diff
@@ -796,6 +797,9 @@ dmu_objset_create_impl(spa_t *spa, dsl_dataset_t *ds, blkptr_t *bp,
 	os->os_phys->os_type = type;
 	if (dmu_objset_userused_enabled(os)) {
 		os->os_phys->os_flags |= OBJSET_FLAG_USERACCOUNTING_COMPLETE;
+		if (dmu_objset_userobjused_enabled(os))
+			os->os_phys->os_flags |=
+			    OBJSET_FLAG_USEROBJACCOUNTING_COMPLETE;
```

Comment:
Sure, I can make it that way. Can you please tell me how to create a zpool without some specific features supported? I tried to create with options "-o features@...=disabled" but it didn't work.